assets/src/features/leaving-page/components/Questionnaire.tsx

```diff
@@ -37,7 +38,10 @@ const Questionnaire: FC<QuestionnaireProps> = ({ onSubmitClick }) => {
   const {isSmartphone} = useSmartphoneViewport();
 
   return (
-    <form aria-label="questionnaire" className="flex flex-col items-center justify-center gap-y-10 sm:gap-y-20">
+    <form aria-label="questionnaire" className={
+      clsx("flex flex-col items-center justify-center",
+      "gap-y-10 sm:gap-y-20",
+      isSmartphone && "mb-36")}>
```

Comment:
https://i.imgflip.com/52xeoo.png

TBH - I thought there is some more elegant solution than putting a fixed element on the button of the page and adding margin to the sibling element. 

I would consider `sticky` element to be a nice solution, but it comes with it's own problems (the element has width of the parent element, not the viewport, doesn't stick to the button etc.).

For simplicity - I used this simple yet sufficient solution.